migrations/migrations/20181126143436-create-user.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
```

Comment:
By convention, we always create our tables with lowercase names only, please change that